### ISSUE TITLE
Could add a parameter to function face_encodings？

### ISSUE THREAD

**Issue by wen1q84** [ageitgey/face_recognition#687]
### Description

Could add a "model" parameter to face_encodings(face_image, known_face_locations=None, num_jitters=1)?
I want to control  input of  function_raw_face_landmarks.

**Comment by ageitgey** [comment]
What is your overall goal?

**Comment by wen1q84** [comment]
when using function face_encodings, I want to use "model" parameter to control function _raw_face_landmarks parameter "model".
Now _raw_face_landmarks default parameter is "small",I want to use "large".

**Comment by ageitgey** [comment]
Why?

**Comment by ageitgey** [comment]
I don't think that would change the results at all. Have you done any tests or think that it would help?

**Comment by haris-ahmed** [comment]
@ageitgey do you think it would produce same output with 5_face_landmarks and  68_face_landmarks?

**Comment by ageitgey** [comment]
Yes
